mlflow/tracking/default_experiment/databricks_notebook_experiment_provider.py

```diff
@@ -8,3 +17,33 @@ def in_context(self):
 
     def get_experiment_id(self):
         return databricks_utils.get_notebook_id()
+
+
+class DatabricksRepoNotebookExperimentProvider(DefaultExperimentProvider):
```

Comment:
Is there a reason this not a separate class file for this new provider?

Reply:
Repo notebooks are kind of "special case" notebooks, didn't seem to warrant its own file. Could move if you feel strongly though